trillian/integration/ct_integration_test.sh

```diff
@@ -31,6 +31,23 @@ RESULT=$?
 set -e
 popd
 
+# Integration test run failed? Clean up and exit if so
+if [[ "${RESULT}" != "0" ]]; then
+  ct_stop_test
+  TO_KILL=()
+
+  exit $RESULT
+fi
+
+# Now run the lifecycle test. This will use the same servers but with a
+# different set of empty logs.
+pushd "${INTEGRATION_DIR}"
+set +e
+go test -v -run ".*LiveLifecycle.*" --timeout=5m ./ --log_config "${CT_LIFECYCLE_CFG}" --admin_server="${RPC_SERVER_1}" --ct_http_servers=${CT_SERVERS} --ct_metrics_servers=${CT_METRICS_SERVERS} --testdata_dir=${GOPATH}/src/github.com/google/certificate-transparency-go/trillian/testdata
```

Comment:
Maybe have a common variable with those arguments that are always going to be shared between the two tests?

Reply:
Done.